src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -273,6 +273,41 @@ file that was distributed with this source code.
                 $(usedInputRef).remove();
                 return true;
             });
+
+            {% set create_url = sonata_admin.field_description.associationadmin.generateUrl('create', sonata_admin.field_description.getOption('link_parameters', {})) %}
```

Comment:
Bonus points if you can find a way to remove the online script to the general javascript file

Reply:
I understand but I don't have enough time now, maybe later

Reply:
Sorry for the typo. I would prefer javascript in the `.js` files instead of inlining them in the templates.

Reply:
Don't have enough time for that.
I don't think getting the JS out in the .js file is a priority.
This feature is awaited for one year, so did you approved it in current state or should we waiting more time ?